powerauth-admin/src/main/resources/application.properties

```diff
@@ -51,6 +51,7 @@ spring.jpa.open-in-view=false
 management.health.ldap.enabled=false
 
 # Monitoring
+management.tracing.sampling.probability=1.0
```

Comment:
@zcgandcomp Should it be turned on by default?

Reply:
yes we can have this turned on by default, but provide a way how to configure it. ie. 1 is default, but there has to be a variable to override it.